server/routes/api/subscriptions.ts

```diff
@@ -27,13 +26,17 @@ router.post("subscriptions.list", auth(), pagination(), async (ctx) => {
   );
 
   const subscriptions = await sequelize.transaction(async (transaction) => {
```

Comment:
There's not really any value to adding transactions to read operations if nothing is getting updated, feel free to remove